core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/dummy/DummyOpExec.scala

```diff
@@ -0,0 +1,23 @@
+package edu.uci.ics.texera.workflow.operators.dummy
+
+import edu.uci.ics.amber.engine.architecture.worker.PauseManager
+import edu.uci.ics.amber.engine.common.InputExhausted
+import edu.uci.ics.amber.engine.common.rpc.AsyncRPCClient
+import edu.uci.ics.texera.workflow.common.operators.OperatorExecutor
+import edu.uci.ics.texera.workflow.common.tuple.Tuple
+
+class DummyOpExec() extends OperatorExecutor {
+
+  override def processTexeraTuple(
+      tuple: Either[Tuple, InputExhausted],
+      input: Int,
+      pauseManager: PauseManager,
+      asyncRPCClient: AsyncRPCClient
+  ): Iterator[Tuple] = {
+    Iterator()
```

Comment:
Add some exceptions here to indicate that it is not supposed to be executed? Same below.